packages/@extension-kid/data-layer/dataLayer.ts

```diff
@@ -73,6 +73,10 @@ export default class DataLayer {
     return makeExecutableSchema({ typeDefs, resolvers });
   }
 
+  getExecutableSchema$(): Observable<GraphQLSchema> {
```

Comment:
There is the concept of a schema, which is a thing that describes your API, but can't actually do anything and there is also the concept of adding the layer of functionality to it calling it an executable schema. The method above is called `getExecutableSchema`, so I would rather make this renaming in a different PR. I think schema makes more sense here, because there are only executable schemas in our DC/OS UI world